doc/src/manual/arrays.md

```diff
@@ -747,15 +750,34 @@ sparse data structure, including structural nonzeros. In order to count the exac
 values that are nonzero, use [`countnz()`](@ref), which inspects every stored element of a sparse
 matrix.
 
-### Sparse matrix constructors
+### Sparse Vector Storage
 
-The simplest way to create sparse matrices is to use functions equivalent to the [`zeros()`](@ref)
-and [`eye()`](@ref) functions that Julia provides for working with dense matrices. To produce
-sparse matrices instead, you can use the same names with an `sp` prefix:
+Sparse vectors are simply constructed by two vectors; one for the stored values and one for
+the corresponding indices. In Julia, sparse vectors have the type
+[`SparseVector{Tv,Ti}`](@ref) where `Tv` is the type of the stored values and `Ti` the
+integer type for the indices. The internal representation is as follows:
+
+```julia
+struct SparseVector{Tv,Ti<:Integer} <: AbstractSparseVector{Tv,Ti}
+    n::Int              # Length of the sparse vector
+    nzind::Vector{Ti}   # Indices of stored values
+    nzval::Vector{Tv}   # Stored values, typically nonzeros
+end
+```
+
+As for [`SparseMatrixCSC`](@ref), the `SparseVector` type can also contain explicitly
+stored zeros, see the manual for [Sparse Matrix Storage](@ref man-csc).
```

Comment:
Perhaps simplify to `... stored zeros. (See [Sparse Matrix Storage](@ref man-csc).)`?